packages/components/src/utils/logger.browser.spec.ts

```diff
@@ -5,6 +5,8 @@ import { mockConsole } from "../tests/utils/logging";
 import { type LogLevel, loggedDeprecations, logger } from "./logger";
 import { type CalciteConfig, clearConfig } from "./config";
 
+const mockComponent = { el: { tagName: "CALCITE-FOO" } } as any;
```

Comment:
Can you try creating a simple test component and passing it in tests ([example](https://github.com/Esri/calcite-design-system/blob/a49438a444528698bb051182de5ad191057a39d1/packages/components/src/controllers/usePreventDocumentScroll.browser.spec.tsx#L7-L16))?